src/App/Resources/AppResources.resx

```diff
@@ -2756,5 +2756,7 @@ Do you want to switch to this account?</value>
   <data name="LogInWithDeviceMustBeSetUpInTheSettingsOfTheBitwardenAppNeedAnotherOption" xml:space="preserve">
     <value>Log in with device must be set up in the settings of the Bitwarden app. Need another option?</value>
   </data>
-
+  <data name="LogInWithDevice" xml:space="preserve">
+    <value>Log In With Device</value>
+  </data>
```

Comment:
⛏️ We need to make this consistent with `Log in initiated` which has its first letter in capital whereas with device has all words capitalized. I guess `Log in initiated` is the right approach but check just in case.